tests/virt/node/general/test_machinetype.py

```
@@ -34,6 +34,23 @@ def vm(request, cluster_cpu_model_scope_function, unprivileged_client, namespace
         yield vm
 
 
+@pytest.fixture()
+def vm_with_non_default_machine_type(
+    request, cluster_cpu_model_scope_function, unprivileged_client, namespace, non_default_machine_type
```

Ruff (0.14.4)

Unused function argument: `cluster_cpu_model_scope_function`

(ARG001)